test/lib/jdk/test/lib/Platform.java

```diff
@@ -260,6 +260,34 @@ public static boolean hasSA() {
         return true;
     }
 
+
+    public static boolean hasPlistEntriesOSX() throws IOException {
```

Comment:
Almost all of this is replicated from `isHardenedOSX()`. I wonder if there is a way to do some sharing while still maintaining separate APIs. Combining them into one API might make it harder to understand the code. Maybe a `launchCodesign()` API that returns the BufferedReader would help.